designs/administrator/future-openshift/runtimeclass-filtering/index.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Filtering by RuntimeClass
+
+## Background
+Kata containers are now being used in OpenShift sandbox. Kata containers are an open-source project that offers light-weight containers with the security of VMs. In OpenShift 4.8 the console team added the Runtime Class field to the Workloads details pages. This allows a user to see what workloads are using the kata runtime on both the Admin and Dev perspective. 
+
+Why is this information important: Deployments using  Kata containers may require additional information and/or policies. Currently in the console the only way to identify which Pods are using the kata runtime class is by going to the details pages (Deployment, Pods, ReplicaSets, etc.).
+
+## Goal
+
+Adding a method to filter by Runtime Class in the Pods list view would give a higher level view of these pods improving observability.
+
+## Designs for a future release
```

Comment:
I think I got slightly confused whether this design was to add a new filter (in the Filter field on workload's list views) based on the Goal above and the name of the design, or was to add the ability to add a column to the workloads list view that contained the Runtime Class as the screens below show (though the Designs for a future release heading made me think maybe these were for a follow on story.)

Reply:
Thanks for the assist @itsptk. I've updated the title and description to clarify the optional column instead of filtering.